.github/workflows/openjdk_14_build.yml

```
@@ -6,37 +6,48 @@ on:
     tags: [ v* ]
 jobs:
   build-and-test:
-    runs-on: ubuntu-latest
+    runs-on: transaction-tools-linux-medium
     timeout-minutes: 180
     env:
       MAVEN_CLI_OPTS: --batch-mode --no-transfer-progress
       TEST_ACCOUNT_ID: ${{ secrets.TEST_ACCOUNT_ID }}
       TEST_PRIVATE_KEY: ${{ secrets.TEST_PRIVATE_KEY }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Harden Runner
+        uses: step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f # v2.10.2
```

```suggestion
        uses: step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f # v2.10.2
```

It's now 2.11.0 if you can update that.